src/test/java/gameframework/motion/MoveStrategyKeyboard8DirTest.java

```diff
@@ -0,0 +1,74 @@
+package gameframework.motion;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import java.awt.Point;
+import java.awt.event.KeyEvent;
+
+import org.junit.Test;
+
+public class MoveStrategyKeyboard8DirTest extends
```

Comment:
if `MoveStrategyKeyboard8Dir extends MoveStrategyKeyboard` then we must have`MoveStrategyKeyboard8DirTest extends MoveStrategyKeyboardTest` as well.